windows/src/engine/kmcomapi/kmcomapi.ridl

```diff
@@ -860,6 +859,50 @@ library keymanapi
     HRESULT _stdcall Uninstall(void);
   };
 
+  [
+    uuid(8D0B3363-E537-437D-AD89-A0B5103F5F64),
+    helpstring("http://help.keyman.com/developer/engine/desktop/10.0/api/IKeymanKeyboardLanguage"),
```

Comment:
Is this going to be a new page? On the `staging-10.0-stable` branch, I see a folder for "IKeymanLanguage"

Reply:
This will be a new page. It's not the same interface as `IKeymanLanguage`